pkg/server/plugin/nodeattestor/gcp/iit.go

```diff
@@ -94,6 +93,39 @@ func (p *IITAttestorPlugin) Attest(stream nodeattestor.Attest_PluginStream) erro
 	return nil
 }
 
+func (p *IITAttestorPlugin) validateAttestationAndExtractIdentityMetadata(stream nodeattestor.Attest_PluginStream, pluginName string) (gcp.ComputeEngine, error) {
```

Comment:
nit: this function uses no state from `p` and can therefore be standalone.

Reply:
It uses the token retriever though that could be passed as an argument.

Reply:
whoops. so it does. feel free to ignore :)